src/merge.rs

```diff
@@ -0,0 +1,138 @@
+use std::marker;
+use std::mem;
+use libc::c_uint;
+
+use {raw, Object, FileFavor};
+use util::Binding;
+use call::Convert;
+
+/// annotated commits, the input to merge and rebase
```

Comment:
Could this be expanded a bit (and use the same sentence-like style of other docs)?